pkg/cmd/openfaas_app_test.go

```diff
@@ -17,3 +22,47 @@ func Test_getValuesSuffix_aarch64(t *testing.T) {
 		t.Errorf("suffix, want: %s, got: %s", want, got)
 	}
 }
+
+var (
+	unexpectedErr         = "failed to merge err: %v, existing flags: %v, set overrides: %v"
+	unexpectedMergeErr    = "error merging flags, expected: %v, got: %v"
+	inconsistentErrString = "inconsistent error return expected: %v, got: %v"
+)
+
+func Test_mergeFlags(t *testing.T) {
+	tests := []struct {
+		flags          map[string]string
+		overrides      []string
+		resultExpected map[string]string
+		errExpected    error
+	}{
+		// positive cases:
+		{map[string]string{}, []string{"b=1"}, map[string]string{"b": "1"}, nil},
```

Comment:
Each case should have a test case name i.e. `Single key with numeric value and no flags` for this one